modules/node_modules/@ncigdc/containers/CaseTable.js

```diff
@@ -72,17 +76,22 @@ export const CaseTableComponent = (props: TTableProps) => {
               <Th rowSpan="2">Primary Site</Th>
               <Th rowSpan="2">Gender</Th>
               <Th rowSpan="2">Files</Th>
-              <Th colSpan="6">Available Files per Data Category</Th>
+              <Th colSpan="7">Available Files per Data Category</Th>
```

Comment:
seeing as you are doing this in the project table, might as well use `Object.keys(DATA_CATEGORIES).length` here :)